www/_posts/2016-03-12-plugin-release.md

```diff
@@ -0,0 +1,74 @@
+---
+layout: post
+author:
+    name: Carlos Santana
+    url: https://twitter.com/csantanapr
+title:  "Plugins Release"
+categories: news
+tags: release plugins
+---
+
+The following plugins were updated today:
+
+* cordova-plugin-camera@2.1.1
+* cordova-plugin-statusbar@2.1.2
+* cordova-plugin-globalization@1.0.3
+* cordova-plugin-splashscreen@3.2.1
+
+
+----
+You can update any plugin by removing it, and then re-adding it.
+
+ E.g. To update your media plugin:
+
+    cordova plugin rm cordova-plugin-camera --save
+    cordova plugin add cordova-plugin-camera@2.1.1 --save
+
+Changes include:
+<!--more-->
+
+cordova-plugin-camera@2.1.1
+* [CB-10825](https://issues.apache.org/jira/browse/CB-10825) android: Always request READ permission for gallery source
```

Comment:
@dpogue weird I didn't catch this I was using VSCode preview and they looked fine, it could a difference in markdown with Github, I fixed, thanks for the review